src/less/components/box/index.less

```diff
@@ -35,4 +35,4 @@
   .flush () {
     padding: @space[flush];
   }
-}
+}
```

Comment:
Try adding a new line here to remove any changes from this file.

You should have a total of **6** `Files changed` for this PR.

Reply:
This shouldn't create any issues.  Merging.